Makefile

```diff
@@ -31,27 +31,43 @@ manager: generate fmt vet
 	go build -o bin/manager ./cmd/manager/main.go
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
-run: install
-	$(KUSTOMIZE) build config/local_run | kubectl apply -n $(NAMESPACE) -f -
+run: install install_crd install_rbac
 	eval $$(scripts/dev/get_e2e_env_vars.py $(cleanup)); \
 	go run ./cmd/manager/main.go
 
 # Install CRDs into a cluster
-install: manifests kustomize
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+install: manifests helm install_crd
+	
+
+install_crd:
+	kubectl apply -f config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml
+
+install_rbac:
+	$(HELM) template -s templates/tests/operator_roles.yaml helm-chart | kubectl  -n $(NAMESPACE) apply -f -
+	$(HELM) template -s templates/tests/database_roles.yaml helm-chart | kubectl  -n $(NAMESPACE) apply -f -
+
+install_manager:
+	kubectl apply -f -n $(NAMESPACE) config/manager/manager.yaml
+
+uninstall_crd:
+	k delete crd mongodbcommunity.mongodbcommunity.mongodb.com
+
+uninstall_rbac: 
+	$(HELM) template -s templates/tests/operator_roles.yaml helm-chart | kubectl delete -f -
```

Comment:
Is it possible to use `helm uninstall` https://helm.sh/docs/helm/helm_uninstall/ for this rather than templating and deleting?